cmd/serverless/main.go

```diff
@@ -481,15 +481,14 @@ func readAPIKeyFromSSM() (string, error) {
 	return "", nil
 }
 
-func stopCallback(cancel context.CancelFunc) {
-	// gracefully shut down any component
+func shutDownGracefully(cancel context.CancelFunc) {
 	cancel()
```

Comment:
Maybe we should leave the shutdown logic to the daemon, and have that control the shutdown order

Reply:
We could do that -- I think this is here because it is called with `defer` in the main `run()` function, so that it always executes, even when we receive a `SIGINT` from the environment. But since none of our other graceful shutdown logic would run in that case, maybe it's better to just consolidate it in the Daemon.